apps/design-land/src/app/image-gallery/image-gallery.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'design-land-image-gallery',
+  templateUrl: './image-gallery.component.html',
+  styleUrls: ['./image-gallery.component.scss']
+})
+export class ImageGalleryComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
I think in the `ngOnInit` we should show an example of defaulting to a particular image when the component loads.